crypto/pem/pvkfmt.c

```diff
@@ -39,7 +39,7 @@ static unsigned int read_ledword(const unsigned char **in)
     ret = *p++;
     ret |= (*p++ << 8);
     ret |= (*p++ << 16);
-    ret |= (*p++ << 24);
+    ret |= ((uint32_t)*p++ << 24);
```

Comment:
So you cast here to uint32_t and then the result is implicitly cast to unsigned int. That's ugly.

Reply:
you are right :)